pkg/build/nodeimage/buildcontext.go

```diff
@@ -135,6 +136,9 @@ func (c *buildContext) buildImage(bits kube.Bits) error {
 		"docker", "commit",
 		// we need to put this back after changing it when running the image
 		"--change", `ENTRYPOINT [ "/usr/local/bin/entrypoint", "/sbin/init" ]`,
+		// remove http proxy setting
```

Comment:
why remove?

Reply:
It's because during my testing, I found that the control panel wasn't able to start up with my built node image, and the built image was carrying the proxy settings. When I removed them, it worked.

Reply:
this does not gives much confidence on the solution proposed :/